tutorials/introduction-to-adnlpmodels/Project.toml

```
@@ -4,5 +4,5 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
 
 [compat]
-ADNLPModels = "0.4"
+ADNLPModels = "0.4, 0.5"
```

```suggestion
ADNLPModels = "0.5"
```
I prefer to use stronger bounds, so there is no doubt that it is using the correct version